examples/vhdl/external_buffer/src/main.c

```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
```

Comment:
This file contains no comment of any purpose or what is achieved.

Reply:
I will add comments.

Reply:
Done.